### PR TITLE
Added conference, camp, developers meeting and review

### PR DESCRIPTION
**Pull request by netzarbeiter**

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -178,3 +178,104 @@ changelog/4.13: https://github.com/contao/contao/blob/4.13/CHANGELOG.md
...
+conference/2011: https://contao.org/de/veranstaltungen/contao-konferenz-2011.html
+conference/2012: https://contao.org/de/veranstaltungen/contao-konferenz-2012.html
+conference/2013: https://contao.org/de/veranstaltungen/contao-konferenz-2013.html
+conference/2014: https://contao.org/de/veranstaltungen/contao-konferenz-2014.html
+conference/2015: https://contao.org/de/veranstaltungen/contao-konferenz-2015.html
+conference/2016: https://contao.org/de/veranstaltungen/contao-konferenz-2016.html
+conference/2017: https://contao.org/de/veranstaltungen/contao-konferenz-2017.html
+conference/2018: https://contao.org/de/veranstaltungen/contao-konferenz-2018.html
+conference/2019: https://contao.org/de/veranstaltungen/contao-konferenz-2019.html
+conference: https://www.contao-konferenz.de/
```
[the remark this answers is not in the document]
Comment (netzarbeiter) [reply redirects.yaml:195]: Unfortunately, the [2019](https://contao.org/de/veranstaltungen/contao-konferenz-2019.html) conference was only created as an event and no data was updated.

[2023](https://contao.org/de/veranstaltungen/contao-konferenz-2023.html) is only a redirect to the conference website.

[the remark this answers is not in the document]
Comment (Toflar) [reply redirects.yaml:196]: Yeah I know, but for the camp, you have also added `camp/2023` as well as `camp/2023`. So my question was, if we should have `conference/2023` too.

[the remark this answers is not in the document]
Comment (netzarbeiter) [reply redirects.yaml:196]: Done

**Review by Toflar — APPROVED** [review]

**Comment by Toflar** [comment]
Thanks @netzarbeiter <3

**Pull request merged**